src/main/java/domain/LottoNumber.java

```diff
@@ -0,0 +1,57 @@
+package domain;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoNumber {
+
+    private static final Map<Integer, LottoNumber> CACHE =
+            IntStream.rangeClosed(
+                            LottoRule.MIN_LOTTO_NUMBER.getValue(),
+                            LottoRule.MAX_LOTTO_NUMBER.getValue())
+                    .boxed()
+                    .collect(Collectors.toMap(
+                            key -> key,
+                            LottoNumber::new));
```

Comment:
```suggestion
                    .collect(Collectors.toMap(key -> key, LottoNumber::new));
```

어느 정도 취향의 영역이긴 하지만 과한 줄바꿈도 가독성을 해칠 수 있습니다. 전체적으로 점검해주세요.

Reply:
감사합니다. 고쳐주신 코드가 더 가독성이 좋은 것 같아요~